src/components/Router/StepComponentMap.js

```diff
@@ -39,14 +40,27 @@ export const enabledDocuments = (steps) => {
   return docTypes ? Object.keys(docTypes).filter((type) => docTypes[type]) : []
 }
 
+const getFaceStep = (steps) => {
+  if (shouldUseVideo(steps)) {
+    return [VideoCapture, VideoConfirm]
+  }
+  return getSelfieStep(steps)
+}
+
+const getSelfieStep = (steps) => {
+  const faceStep = steps.filter(step => step.type === "face")[0]
+  if (faceStep.options.useWebcam) {
```

Comment:
After a conversation with @kopijunkie, we noticed that you are relying on `useWebcam` to return the Selfie components. We have tickets in our backlog to remove reference to this option in the near future, so we think it's better to take a different approach. Might be worth discussing this offline with @kopijunkie

Reply:
I've fixed the merge conflict from my code changes that is already in `development`. So to resolve this, it should just be a simple change here to update the option/parameter that is being used in the check at line 52. @anuragajwani you can refer to what I've done for handling the Video step components (let me know if you need me to walk through what I've done).